phoenix-core/src/main/java/org/apache/phoenix/mapreduce/util/PhoenixConfigurationUtil.java

```diff
@@ -191,6 +190,12 @@
     // provide an absolute path to inject your multi input mapper logic
     public static final String MAPREDUCE_MULTI_INPUT_MAPPER_TRACKER_CLAZZ = "phoenix.mapreduce.multi.mapper.tracker.path";
 
+    // provide control to whether or not handle MR snapshot restore on phoenix side or handled by caller externally
```

Comment:
Also could you please add couple of lines on what does it mean to manage internally vs externally ?